src/explore-education-statistics-frontend/src/modules/glossary/GlossaryIndexPage.tsx

```diff
@@ -15,6 +15,10 @@ function GlossaryIndexPage() {
             Browse our A to Z list of definitions for terms used across
             education statistics and data.
           </p>
+          <p className="govuk-body">
```

Comment:
I'm wondering if we should be adding this as it wasn't in the prototype created by Fred.

Reply:
I added this line of text at Laura's request